pants

```diff
@@ -195,6 +195,10 @@ function determine_default_python_exe {
     if [[ "$("${interpreter_path}" --version 2>&1 > /dev/null)" == "pyenv: python${version}"* ]]; then
       continue
     fi
+    # Check if a version is shimmed by asdf but not configured.
+    if ! ${interpreter_path} --version >/dev/null 2>&1; then
```

Comment:
My shell-fu is not strong, but I think this needs quoting, in case the path has spaces in it?